falcon/request_helpers.py

```diff
@@ -38,6 +42,29 @@ def fget(self):
     return property(fget)
 
 
+def parse_etags(etag_str):
+    """
+    Parse a string of ETags given in an If-Match or If-None-Match header as
```

Comment:
I am next to hopeless at properly using articles, but I believe we should use **the** here, if we know which header it is.
So :arrow_right: **the** ``If-Match``... etc